gitlab/1.9/README.md

```diff
@@ -0,0 +1,227 @@
+# How to use GitLab on DC/OS
+
+[Gitlab](https://gitlab.com) an open source developer tool that allows you to host git repositories, review code, track issues, host Docker images and perform continuous integration.
+
+Using GitLab on DC/OS allows you to co-locate all of the tools you need for developers on one easy to manage cluster. Just as with any Universe package, you can robustly install several side by side instances of GitLab to provide segregated instances for each of your development teams. Alternatively, you can just as easily install GitLab in a highly available configuration that many teams use concurrently.
+
+![Continuous Delivery with GitLab](img/gitlab-cd.png)
+
+This quickstart installation uses the single node version of GitLab that includes an installation of Postgres and Redis in the same container.
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down. We recommend checking out the production installation instructions on alternative options and considerations.
+
+- Estimated time for completion: up to 90 minutes
+- Target audience:
+ - Operators
+ - Application admins
+ - Quality/Release engineers
+ - CI/CD admins
+ - Devops Engineers
+- Scope: You'll learn how to install GitLab and how to use it to build and deploy a Docker image on Marathon.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Setting up GitLab](#setting-up-gitlab)
+- [Setting up the Repository](#setting-up-the-repository)
+- [Set up the Pipeline](#set-up-the-pipeline)
+- [Resources](#resources)
+
+## Prerequisites
+
+- DC/OS 1.9 or later
+- [Marathon-lb](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/usage/) must be installed and running
+- [Jenkins](https://docs.mesosphere.com/1.9/usage/service-guides/jenkins/) must be installed and running
+- An available hostname configured to point to the public agent(s) where Marathon-lb is running (e.g. `gitlab-test.mesosphere.com`)
+- Either set up appropriate certificates on each of your hosts, or configure each of your hosts to use your hostname (i.e. `gitlab-test.mesosphere.com`) as a [Docker insecure registry](https://docs.docker.com/registry/insecure/).
```

Comment:
Would love clarification here on installation. Do you recommend using docker registry dcos package?